openapi.yml

```
@@ -1131,6 +1131,101 @@ paths:
       description: Update a favorite with new FavoriteDetails
       requestBody:
         $ref: "#/components/requestBodies/CreateFavoriteRequest"
+  "/api/v1/target-group-deployments/{id}":
+    get:
+      summary: Your GET endpoint
```

Prior to merging, we need to update the descriptions on the APIs and Models

The ids of the routes look good, nice work

```suggestion
      summary: Your GET endpoint
```

---

Have just added some preliminary descriptions to each end point